shared/src/models/map-image.ts

```diff
@@ -15,12 +15,16 @@ export class MapImage {
     @Type(() => ImageProperties)
     public readonly image: ImageProperties;
 
+    @IsBoolean()
+    public readonly locked: boolean;
+
     /**
      * @deprecated Use {@link create} instead
      */
-    constructor(topLeft: Position, image: ImageProperties) {
+    constructor(topLeft: Position, image: ImageProperties, locked?: boolean) {
         this.position = topLeft;
         this.image = image;
+        this.locked = locked ?? false;
```

Comment:
Is there a special reason you have made it optional?
I believe it is rather untypical in the models to have a property optional with a default value.

Btw. you would probably rather do it like this:

```suggestion
    constructor(topLeft: Position, image: ImageProperties, locked = false) {
        this.position = topLeft;
        this.image = image;
        this.locked = locked;
```

Reply:
We had the hope that making it optional would be an alternative to writing a migration, but since you've said that we need the migration anyways, we're going to change this.

Reply:
There is a migration now ([8f2ce60](https://github.com/hpi-sam/digital-fuesim-manv/pull/545/commits/8f2ce60b9dc2e73956792294684b65a352cbbf4e))